doc/CONFIGURATION.md

```diff
@@ -201,6 +201,31 @@ In its default configuration, there is no maximum on the size of objects Mountpo
 
 To increase the maximum object size for writes, use the `--part-size` command-line argument to specify a maximum number of bytes per part, which defaults to 8 MiB. The maximum object size will be 10,000 multiplied by the value you provide for this argument. Even with multipart upload, S3 allows a maximum object size of 5 TiB, and so setting this argument higher than 524.3 MiB will not further increase the object size limit.
 
+### Mounting an S3 bucket after reboot
+
+A tracking issue is open for _fstab_ support: [#44](https://github.com/awslabs/mountpoint-s3/issues/44).
+
+Until this support is implemented, we recommend using a system manager like systemd to monitor the mount process and mount during boot.
+Below is an example of a systemd unit for mountpoint-s3.
+Replace `/home/ec2-user/s3-bucket-mount` and `DOC-EXAMPLE-BUCKET` with your mount directory and S3 bucket.
+
+```ini
+[Unit]
+Description=Mountpoint for Amazon S3 mount
+Wants=network.target
+AssertPathIsDirectory=/home/ec2-user/s3-bucket-mount
+
+[Service]
+Type=forking
+User=ec2-user
+Group=ec2-user
+ExecStart=/usr/bin/mount-s3 DOC-EXAMPLE-BUCKET /home/ec2-user/s3-bucket-mount
+ExecStop=/usr/bin/fusermount -u /home/ec2-user/s3-bucket-mount
+
+[Install]
+WantedBy=default.target
```

Comment:
I'm no systemd guru, but should this be something like `remote-fs.target` instead?

Reply:
Oh nice! Me neither, but that does sound about right. I'll try it and update.

Reply:
Works well! Tested on an ARM AL2023 instance, works fine before and after reboot. I think it makes sense.

This is the graph it builds (with some boring long bits removed):

```
efault.target
○ ├─display-manager.service
○ ├─systemd-update-utmp-runlevel.service
● └─multi-user.target
●   ├─amazon-ssm-agent.service
●   ├─atd.service
●   ├─auditd.service
●   ├─chronyd.service
●   ├─docker.service
<SNIP>
●   ├─sshd.service
●   ├─sssd.service
●   ├─sysstat.service
●   ├─systemd-ask-password-wall.path
●   ├─systemd-homed.service
●   ├─systemd-logind.service
●   ├─systemd-networkd.service
○   ├─systemd-update-utmp-runlevel.service
●   ├─systemd-user-sessions.service
○   ├─update-motd.service
●   ├─basic.target
●   │ ├─-.mount
○   │ ├─rpmdb-rebuild.service
●   │ ├─tmp.mount
●   │ ├─paths.target
●   │ ├─slices.target
●   │ │ ├─-.slice
●   │ │ └─system.slice
●   │ ├─sockets.target
●   │ │ ├─dbus.socket
●   │ │ ├─docker.socket
●   │ │ ├─sssd-kcm.socket
●   │ │ ├─systemd-coredump.socket
<SNIP>
●   │ │ └─systemd-userdbd.socket
●   │ ├─sysinit.target
●   │ │ ├─dev-hugepages.mount
<SNIP>
●   │ │ ├─systemd-update-utmp.service
●   │ │ ├─cryptsetup.target
●   │ │ ├─integritysetup.target
●   │ │ ├─local-fs.target
●   │ │ │ ├─-.mount
●   │ │ │ ├─boot-efi.automount
●   │ │ │ ├─systemd-fsck-root.service
●   │ │ │ ├─systemd-remount-fs.service
●   │ │ │ └─tmp.mount
●   │ │ ├─swap.target
●   │ │ └─veritysetup.target
●   │ └─timers.target
●   │   ├─fstrim.timer
●   │   ├─logrotate.timer
●   │   ├─systemd-tmpfiles-clean.timer
●   │   └─update-motd.timer
●   ├─cloud-init.target
●   │ ├─cloud-config.service
●   │ ├─cloud-final.service
●   │ ├─cloud-init-local.service
●   │ └─cloud-init.service
●   ├─getty.target
●   │ ├─getty@tty1.service
●   │ └─serial-getty@ttyS0.service
●   ├─nfs-client.target
○   │ ├─auth-rpcgss-module.service
●   │ ├─rpc-statd-notify.service
●   │ └─remote-fs-pre.target
●   ├─remote-cryptsetup.target
●   └─remote-fs.target
●     ├─mountpoint-s3.service
●     └─nfs-client.target
○       ├─auth-rpcgss-module.service
●       ├─rpc-statd-notify.service
●       └─remote-fs-pre.target
```